hvpy/api.py

```diff
@@ -1,13 +1,76 @@
 __all__ = ["fake_api_call"]
 
+from datetime import datetime
+from typing import Optional
+import requests
+from pydantic import BaseModel, Field
 
-def fake_api_call():
+
+class InputParameters(BaseModel):
+    date: datetime
+    sourceId: int
+    jpip: Optional[bool] = False
+    Json: Optional[bool] = Field(False, alias="json")
+
+
+def parse_response(response: requests.Response, output_parameters: str):
     """
     _summary_
 
+    Parameters
+    ----------
+    response: ?
+        _description_
+    output_parameters: ?
+        _description_
```

Comment:
"Expected response type from the API, used to convert the API response into the given type."